apps/frontend/src/modules/surveys/SurveyProvider.tsx

```diff
@@ -165,6 +165,7 @@ export const SurveyProvider = ({ children }: SurveyProviderProp) => {
                                         },
                                         ...existingSurveysRefs.edges,
                                     ],
+                                    totalCount: existingSurveysRefs.totalCount + 1,
```

Comment:
@AfeezGL What's the point of the extra 1 added?

Reply:
At the footer of the survey list table we are using the `totalCount` to show the total surveys the project has, the total count was not being updated before when we create a new survey. This led to a behavior where if you have one survey in a project and you land on the survey list page, you see `1 survey` at the bottom of the table but when you create a new survey and go back to the survey list page, you see `1 survey` at the bottom of the table even though you can see two surveys in the table.